build.gradle.kts

```diff
@@ -108,7 +108,7 @@ tasks.jar {
 
 java {
   toolchain {
-    languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
Is this change intended?

Reply:
Oops -- no, that was to get it to build on my machine.  I will remove.